File: lib/containers/horizontal_dock_container.dart
part of dock_spawn;

class HorizontalDockContainer extends SplitterDockContainer  {

  HorizontalDockContainer(DockManager dockManager, List<IDockContainer> childContainers) 
      : super(getNextId("horizontal_splitter_"), dockManager, childContainers) {
    containerType = "horizontal";
  }
  
  bool get stackedVertical {
    return false;
  }
}

File: js/lib/containers/HorizontalDockContainer.js
import "SplitterDockContainer";

dockspawn.HorizontalDockContainer = function(dockManager, childContainers)
{
    this.stackedVertical = false;
    dockspawn.SplitterDockContainer.call(this, getNextId("horizontal_splitter_"), dockManager, childContainers);
    this.containerType = "horizontal";
};
dockspawn.HorizontalDockContainer.prototype = new dockspawn.SplitterDockContainer();
dockspawn.HorizontalDockContainer.prototype.constructor = dockspawn.HorizontalDockContainer;
